shared/src/packages/meditation/libs/validation-schemas/create-meditation-entry-request.validation-schema.ts

```diff
@@ -0,0 +1,40 @@
+import joi from 'joi';
+
+import { ContentType } from '#libs/enums/enums.js';
+
+import { MeditationEntryValidationMessage } from '../enums/enums.js';
+import { type MeditationEntryCreateFormDataDto } from '../types/types.js';
+
+const createMeditationEntryRequest = joi.object<
+  MeditationEntryCreateFormDataDto,
+  true
+>({
+  name: joi
+    .object({
+      value: joi.string().required().messages({
+        'any.required': MeditationEntryValidationMessage.NAME_REQUIRED,
+        'string.emplty': MeditationEntryValidationMessage.NAME_REQUIRED,
+      }),
+      mimetype: joi.string().valid(ContentType.TEXT).required().messages({
+        'any.only': MeditationEntryValidationMessage.MPEG_REQUIRED,
+      }),
+    })
+    .unknown(true)
+    .required()
+    .messages({
+      'object.base': MeditationEntryValidationMessage.NAME_REQUIRED,
+    }),
+  file: joi
+    .object({
+      mimetype: joi.string().valid(ContentType.MPEG).required().messages({
+        'any.only': MeditationEntryValidationMessage.MPEG_REQUIRED,
+      }),
+    })
+    .unknown(true)
+    .required()
+    .messages({
+      'object.base': MeditationEntryValidationMessage.FILE_REQUIRED,
+    }),
+});
```

Comment:
Actually, we have a prehandler fileUploadPlugin and it can handle it

Reply:
@v0ldemar01 I was asked to manage validation also on the backend side. I have to do extra type validation only for audio and actually check the received name

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/e59b73bb-cfb0-4921-a9e2-d7410c37c201)